src/compiler/msvc.rs

```diff
@@ -1334,8 +1339,8 @@ mod test {
     #[test]
     fn test_parse_arguments_too_many_inputs() {
         assert_eq!(
-            CompilerArguments::CannotCache("multiple input files", None),
-            parse_arguments(ovec!["-c", "foo.c", "-Fofoo.obj", "bar.c"])
+            CompilerArguments::CannotCache("multiple input files", Some("bar.c".to_string())),
```

Comment:
```suggestion
            CompilerArguments::CannotCache("multiple input files", Some("[bar.c]".to_string())),
```

Reply:
Changed and added a new test for a single and multiple multiple-input files (like in `gcc.rs`).